src/mixins/date-mixin.ts

```diff
@@ -16,157 +16,130 @@ export default class DateMixin extends Vue {
   readonly currentDate!: string
 
   /**
-   * Converts a JavaScript date object to a simple date string (YYYY-MM-DD) in Pacific timezone.
+   * Converts a Date object to a date string (YYYY-MM-DD) in Pacific timezone.
```

Comment:
This looks like a lot of changes but it's (mostly) a copy of the date mixin from our other projects.

At some point maybe we can move this to the shared components library!